### PR TITLE
chore: rename package to @redocly/cli

### PR DESCRIPTION
**Pull request by RomanHotsiy**
## What/Why/How?

The CLI tool is now more than just OpenAPI toolset. It also have redocly-specific commands like `login` and `push`. We decided to rename the `@redocly/openapi-cli` package to simply `@redocly/cli` in npm.

We keep `@redocly/openapi-core` package name unchanged.

The previous package will be marked as deprecated in npm but will continue to work just fine.

This PR has a lot of docs updates so I need @skadinna or @adamaltman approval.

Todo after merging this PR:

- [ ] Mark old name as deprecated in npm
- [ ] Rename GitHub repository to `redocly-cli`

Todo later (may be in cooldown):

- [ ] rename docker image name appropriatelly
- [ ] update Redocly Workflows deployment scripts to use new names for consistency

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ tocMaxDepth: 2
 
 - Fixed resolving scalar values in asserts.
 
+### Deprecation
```
Comment (skadinna) [inline docs/changelog.md:19]: question: should we add an admonition here that we will still support the old "openapi" naming in commands? In general, we should announce this change somewhere, add a note that we've updated all references to "OpenAPI CLI" in the docs to "Redocly CLI", and any other relevant info.

File: docs/commands/bundle.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 API definitions can grow and become difficult to manage, especially if several teams are collaborating on them. It's a good practice to maintain the reusable parts as separate files, and include them in the main (root) API definition by referencing them with `$ref`. However, most OpenAPI tools don't support that multi-file approach, and require a single-file API definition.
 
-Redocly OpenAPI CLI can help you combine separate API definition files into one. The `bundle` command pulls the relevant parts of an API definition into a single file output in JSON or YAML format.
+Redocly Redocly CLI can help you combine separate API definition files into one. The `bundle` command pulls the relevant parts of an API definition into a single file output in JSON or YAML format.
```
Comment (skadinna) [inline docs/commands/bundle.md:7]: duplicated

Comment (skadinna) [inline docs/commands/bundle.md:7]:
```suggestion
Redocly CLI can help you combine separate API definition files into one. The `bundle` command pulls the relevant parts of an API definition into a single file output in JSON or YAML format.
```

File: docs/commands/index.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 tocMaxDepth: 2
 ---
 
-# OpenAPI CLI commands
+# Redocly CLI commands
 
-Redocly OpenAPI CLI currently supports the following commands:
+Redocly Redocly CLI currently supports the following commands:
```
Comment (skadinna) [inline docs/commands/index.md:7]:
```suggestion
Redocly CLI currently supports the following commands:
```

File: docs/commands/join.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ The `join` command supports OpenAPI 3.x definitions only.
 
 Maintainers of multiple API definitions can benefit from storing each endpoint as a standalone API definition file. However, this approach is not supported by the majority of OpenAPI tools, as they require a single API definition file.
 
-With Redocly OpenAPI CLI, you can solve this problem by using the `join` command that can combine two or more API definition files into a single one.
+With Redocly Redocly CLI, you can solve this problem by using the `join` command that can combine two or more API definition files into a single one.
```
Comment (skadinna) [inline docs/commands/join.md:18]:
```suggestion
With Redocly CLI, you can solve this problem by using the `join` command that can combine two or more API definition files into a single one.
```

File: docs/commands/lint.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions. This helps you avoid bugs and make API definitions more consistent.
+Redocly Redocly CLI can identify and report on problems found in OpenAPI definitions. This helps you avoid bugs and make API definitions more consistent.
```
Comment (skadinna) [inline docs/commands/lint.md:5]:
```suggestion
Redocly CLI can identify and report on problems found in OpenAPI definitions. This helps you avoid bugs and make API definitions more consistent.
```

File: docs/guides/lint.md [hunk 6]
```diff
@@ -245,7 +245,7 @@ Be sure to document those options for your users.
 
 ### Different OpenAPI versions
 
-Redocly OpenAPI CLI supports OpenAPI versions 2.0, 3.0, and 3.1. Most of the time you will use one of them. However, you may need to configure different rules based on the version. You can do that by using additional configuration sections:
+Redocly Redocly CLI supports OpenAPI versions 2.0, 3.0, and 3.1. Most of the time you will use one of them. However, you may need to configure different rules based on the version. You can do that by using additional configuration sections:
```
Comment (skadinna) [inline docs/guides/lint.md:248]:
```suggestion
Redocly CLI supports OpenAPI versions 2.0, 3.0, and 3.1. Most of the time you will use one of them. However, you may need to configure different rules based on the version. You can do that by using additional configuration sections:
```

File: docs/quickstart.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# Redocly OpenAPI CLI quickstart guide
+# Redocly Redocly CLI quickstart guide
```
Comment (skadinna) [inline docs/quickstart.md:1]:
```suggestion
# Redocly CLI quickstart guide
```

File: docs/quickstart.md [hunk 8]
```diff
@@ -217,7 +217,7 @@ You should get this response:
 
 ```bash
 Using Redoc community edition.
-Login with openapi-cli login or use an enterprise license key to preview with the premium docs.
+Login with redocly login or use an enterprise license key to preview with the premium docs.
```
Comment (skadinna) [inline docs/quickstart.md:220]:
```suggestion
Login with `redocly login` or use an enterprise license key to preview with the premium docs.
```

**Review by tatomyr — APPROVED** [review]

File: docs/changelog.md (hunk 1 above)
Comment (RomanHotsiy) [reply docs/changelog.md:19]: I think we should.

Comment (RomanHotsiy) [reply docs/changelog.md:19]: @skadinna can you help with it?

**Review by AlexVarchuk — APPROVED** [review]

**Comment by minalabib223** [comment]
في جمعة، ٦ مايو، ٢٠٢٢ في ١١:٣٣ ص، كتب Mina Labib ***@***.***>:

**Comment by minalabib223** [comment]
في جمعة، ٦ مايو، ٢٠٢٢ في ١١:٣٤ ص، كتب minalabib223 ***@***.***

File: README.md [hunk 9]
```diff
@@ -27,25 +27,25 @@ Currently, @redocly/openapi-cli supports these features:
 
 ## What makes this tool different
 
-Unlike other OpenAPI linters, `@redocly/openapi-cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how linters for programming languages work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators.
+Unlike other OpenAPI linters, `@redocly/cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how linters for programming languages work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators.
```
Comment (adamaltman) [inline README.md:30]: idea: We can improve this section (not related to this PR, but didn't want to lose the idea). Something like:

- **It's faster** (uses a type tree similar to how linters for programming languages work)
- **It's highly configurable** (comes with a slew of commands and rules)
- **It's more extensible** (architected for custom plugins and types)

What do you think @skadinna ?

**Comment by minalabib223** [comment]
في جمعة، ٦ مايو، ٢٠٢٢ في ٤:٢٢ م، كتب Ivana Isadora Devcic <
***@***.***>:

**Comment by minalabib223** [comment]
في جمعة، ٦ مايو، ٢٠٢٢ في ٤:٣٨ م، كتب minalabib223 ***@***.***

File: docs/changelog.md (hunk 1 above)
Comment (skadinna) [reply docs/changelog.md]: I made some changes here: https://github.com/Redocly/openapi-cli/pull/672/commits/22bf9d612d350d1d4d60ba6f4e4ca87baac0c193

File: README.md (hunk 9 above)
Comment (adamaltman) [reply README.md:30]: I opened a PR already: https://github.com/Redocly/redocly-cli/pull/678

**Pull request merged**